modules/release-notes/ref-release-notes-known-issues.adoc

```diff
@@ -4,6 +4,23 @@
 
 This section lists known issues in {product} {product-version}.
 
+[id="known-issue-rhidp-5344"]
+== Multi-Attached error for Volume (PVC)
```

Comment:
This has been fixed in 1.5.0 (purpose of the breaking change which reverted to making the PVC ephemeral by default in the Helm Chart). So this is no longer a known issue IMO.

Reply:
The Jira issue is still in the Backlog state. If this has been fixed, should we update the status and close it? Here’s the link for reference: [RHIDP-5344](https://issues.redhat.com/browse/RHIDP-5344).

Reply:
https://issues.redhat.com/browse/RHIDP-5344 affects 1.4, but has been fixed in 1.5. So it would be a known issue for 1.4, but not for 1.5.
How was this actually picked up in the release notes here? I didn't see anything related to 1.5 on the JIRA itself.

Reply:
As this issue is not yet resolved, we have carried it forward under the known issues from the previous release, following our standard process. We will update it under the fixed issues section if it has been resolved. Could you please share the Jira ticket for version 1.5 where this issue is marked as fixed?

Reply:
> Could you please share the Jira ticket for version 1.5 where this issue is marked as fixed?

This is the original issue where this was fixed in 1.5.0: https://issues.redhat.com/browse/RHIDP-5342
Then https://issues.redhat.com/browse/RHIDP-5344 was created, then closed as duplicate of RHIDP-5432, then reopened again to be documented as a known issue in 1.4

Reply:
RHIDP-5342 has already been added as a fixed issue in the RNs:
![image](https://github.com/user-attachments/assets/43cd0ca1-423f-45cc-82d6-8fd9e08bb623)
So, I'll remove the known issue for it.
Thanks Armel